### DISCUSSION TITLE
use bootstrap bundle min js mobile navbar can't close

### DISCUSSION THREAD

**Discussion by DanielKong94** [twbs/bootstrap#31870] [General]
using v5 alpha 2 bootstrap. early is using bootstrap.bundle.min.js, the mobile version navbar can't close. the "show" class keep showing. when i change the bundle js file to bootsrap.min.js this issue is fix. may i know any one have this issue before?

**Comment by mdo** [comment]
_You should always include a **live demo** of the problem. Create a reduced test case via [CodePen](https://codepen.io/) or [JS Bin](https://jsbin.com/) and report back with your link, Bootstrap version, and specific browser and operating system details._

**Reply by DanielKong94** [reply]
i use codepen run the cdn no problem.



my bootstrap version is 5.0.0-alpha2
browser is google chrome
operating system is windows

**Reply by DanielKong94** [reply]
https://gph.is/g/aNJ2yw1
this is my gif demo at my computer
